docs/guides/index.md

```diff
@@ -1,6 +1,15 @@
 # In-depth explanations
 
-To learn about using `magicgui` in napari, see the {ref}`magicgui` guide.
+```{note}
+These pages describe advanced usage and how napari works internally. If you are
+just getting started, check out our tutorials or how-to guides instead.
```

Comment:
@melissawm could you add links for these here?

Reply:
Done!